src/main/java/chess/Board.java

```diff
@@ -0,0 +1,25 @@
+package chess;
+
+import pieces.Pawn;
+
+import java.util.ArrayList;
+
+public class Board {
+    private final ArrayList<Pawn> pawns;
```

Comment:
지금 학습하실 내용은 아니긴 한데, `java.util.collection` 패키지에 있는 클래스들을 활용할 때는,
구현체 클래스로 직접 타입을 지정해주기보단 [상위 인터페이스](https://en.wikipedia.org/wiki/Java_collections_framework#Architecture) 에 해당하는 타입을 지정해주는 것이 좋습니다. 

지금 단계에서 조금 이해하시기 어려운 내용일 수도 있어서, 자세한 내막(?) 은 설명드리지 않겠지만 학습 초기에 잘못 들 수 있는 습관이어서 지금 리뷰해 드립니다. 

```suggestion
    private final List<Pawn> pawns;

Reply:
오우 저 마침 이것 때문에 계속 고민 중이어서 [다른 팀원 분에게도 여쭤보고 했는데](https://github.com/codesquad-members-2021/java-chess/pull/44#discussion_r573288974), 자세한 내막(?)도 알고 싶군요. 
"콜렉션을 사용할 땐 상위 인터페이스에 해당하는 타입을 지정한다." 잘 새기고 있겠습니다!

Reply:
@wheejuni 안녕하세요! 제인입니다🥰  소중한 피드백 감사드립니다🙏
ArrayList에서 List로 타입을 변경하였고, 링크해주신 부분과 [추가 링크](https://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n)를 통해 이유를 학습하였습니다. 궁금했던 부분인데 감사합니다ㅎㅎ